src/views/Transactions.vue

```diff
@@ -1,34 +1,39 @@
 <template>
     <div class="transaction transaction_list" ref="txListElement">
-      <md-list class="custom-list md-triple-line md-transparent">
-        <md-list-item v-for="(transaction) in transactions" :key="transaction.id" style="cursor:pointer">
-          <md-avatar>
-            <md-icon>{{ transaction.direction === 'from' ? 'flight_takeoff' : 'flight_land' }}</md-icon>
-          </md-avatar>
 
-          <div class="md-list-text-container" v-on:click="goToTransaction(transaction.id)">
-            <div>
-              {{ displayName(transaction) }}
-              <span class="partner_display_name">{{ formatPartnerAddress(transaction) }}</span>
+      <template v-if="isFulfilled">
+        <md-list v-if="hasTransactions" class="custom-list md-triple-line md-transparent">
+          <md-list-item v-for="(transaction) in transactions" :key="transaction.id" style="cursor:pointer">
+            <md-avatar>
+              <md-icon>{{ transaction.direction === 'from' ? 'flight_takeoff' : 'flight_land' }}</md-icon>
+            </md-avatar>
+
+            <div class="md-list-text-container" v-on:click="goToTransaction(transaction.id)">
+              <div>
+                {{ displayName(transaction) }}
+                <span class="partner_display_name">{{ formatPartnerAddress(transaction) }}</span>
+              </div>
+              <span>{{ $formatAmount(transaction.amount, crypto) }} {{crypto}}</span>
+              <p>{{ $formatDate(transaction.timestamp) }}</p>
             </div>
-            <span>{{ $formatAmount(transaction.amount, crypto) }} {{crypto}}</span>
-            <p>{{ $formatDate(transaction.timestamp) }}</p>
-          </div>
 
-          <md-button
-            class="md-icon-button md-list-action"
-            v-if="showChat"
-            @click="openChat(transaction)"
-          >
-            <md-icon>{{ hasMessages(transaction) ? "chat" : "chat_bubble_outline" }}</md-icon>
-          </md-button>
+            <md-button
+              class="md-icon-button md-list-action"
+              v-if="showChat"
+              @click="openChat(transaction)"
+            >
+              <md-icon>{{ hasMessages(transaction) ? "chat" : "chat_bubble_outline" }}</md-icon>
+            </md-button>
+
+            <md-divider class="md-inset"></md-divider>
+          </md-list-item>
+          <md-list-item class="transaction_list__loader" v-if="isLoading">
+            <inline-spinner />
+          </md-list-item>
+        </md-list>
 
-          <md-divider class="md-inset"></md-divider>
-        </md-list-item>
-        <md-list-item class="transaction_list__loader" v-if="isLoading">
-          <inline-spinner />
-        </md-list-item>
-      </md-list>
+        <h3 v-else class="md-headline">{{ $t('transaction.no_transactions') }}</h3>
```

Comment:
In an `if ... else ...` expression the shorter branch should go first for better readability.